tests/Jellyfin.Naming.Tests/Subtitles/SubtitleParserTests.cs

```diff
@@ -1,39 +1,46 @@
-﻿using Emby.Naming.Common;
+﻿using System;
+using Emby.Naming.Common;
 using Emby.Naming.Subtitles;
 using Xunit;
 
 namespace Jellyfin.Naming.Tests.Subtitles
 {
     public class SubtitleParserTests
     {
-        private SubtitleParser GetParser()
-        {
-            var options = new NamingOptions();
-
-            return new SubtitleParser(options);
-        }
-
-        [Fact]
-        public void TestSubtitles()
-        {
-            Test("The Skin I Live In (2011).srt", null, false, false);
-            Test("The Skin I Live In (2011).eng.srt", "eng", false, false);
-            Test("The Skin I Live In (2011).eng.default.srt", "eng", true, false);
-            Test("The Skin I Live In (2011).eng.forced.srt", "eng", false, true);
-            Test("The Skin I Live In (2011).eng.foreign.srt", "eng", false, true);
-            Test("The Skin I Live In (2011).eng.default.foreign.srt", "eng", true, true);
-            Test("The Skin I Live In (2011).default.foreign.eng.srt", "eng", true, true);
-        }
+        private readonly NamingOptions _namingOptions = new NamingOptions();
 
-        private void Test(string input, string language, bool isDefault, bool isForced)
+        [Theory]
+        [InlineData("The Skin I Live In (2011).srt", null, false, false)]
+        [InlineData("The Skin I Live In (2011).eng.srt", "eng", false, false)]
+        [InlineData("The Skin I Live In (2011).eng.default.srt", "eng", true, false)]
+        [InlineData("The Skin I Live In (2011).eng.forced.srt", "eng", false, true)]
+        [InlineData("The Skin I Live In (2011).eng.foreign.srt", "eng", false, true)]
+        [InlineData("The Skin I Live In (2011).eng.default.foreign.srt", "eng", true, true)]
+        [InlineData("The Skin I Live In (2011).default.foreign.eng.srt", "eng", true, true)]
+        public void SubtitleParser_ValidFileNames_Parses(string input, string language, bool isDefault, bool isForced)
```

Comment:
```suggestion
        public void SubtitleParser_WithValidFileName_ParsesCorrectly(string input, string language, bool isDefault, bool isForced)
```

Reply:
Update the comment above. I think the context should be singular instead of plural.